packages/react/index.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import '@clr/core/global.min.css';
+import '@clr/core/styles/module.typography.min.css';
+import '@clr/core/styles/module.layout.min.css';
```

Comment:
I believe with the latest version on master these are the imports needed. Typography and layout are now in global by default.

```scss
@import '~normalize.css/normalize.css';
@import '~@clr/core/styles/global.min';
@import '~@clr/city/css/bundles/default.min';
```

Reply:
Yup, making a note to update these after rebase.